### ISSUE TITLE
Emissive Materials are not applied when rendering entities in the player's inventory

### ISSUE THREAD

**Issue by Sollace** [grondag/canvas#207]
Version: Canvas-mc116-1.0.1267
Mc: 1.16.4 (dev environment)
Fabric: 0.30.0+1.16
Loader: 0.10.6+build.214
No other mods are included that mess with rendering

Glowy:
![image](https://user-images.githubusercontent.com/6429283/107490027-02204a80-6b92-11eb-90ba-3f1a98f159db.png)

Not Glowy:
![image](https://user-images.githubusercontent.com/6429283/107489882-d43b0600-6b91-11eb-946b-f3002a4be9bf.png)

I did check and my code does correctly set the material when rendering in the inventory.

**Comment by Dolphin2Point1** [comment]
This can be classified as #189, because it only applies shaders to worlds.

**Comment by spiralhalo** [comment]
Yeah, Canvas only apply post processing effects to the world. Material shader is still applied to GUI items but not post processing like bloom. I dunno if it can be a feature or not, could be a performance issue or design restrictions such as not letting buggy shader mess with the rest of the GUI.

**Comment by Dolphin2Point1** [comment]
I doubt that it would lag because there are not that many pixels in that very small window

**Comment by grondag** [comment]
Mojang's GUI rendering is problematic, so would not get too optimistic about this.
